metricbeat/module/prometheus/remote_write/_meta/docs.asciidoc

```diff
@@ -20,3 +31,15 @@ Also consider using secure settings for the server using TLS/SSL as shown
   ssl.key: "/etc/pki/server/cert.key"
   port: "9201"
 ------------------------------------------------------------------------------
+
+and on Prometheus side:
+
+["source","yaml",subs="attributes"]
+------------------------------------------------------------------------------
+remote_write:
+  - url: "https://localhost:9201/write"
+    tls_config:
+        cert_file: "/etc/prometheus/my_key.pem"
+        key_file: "/etc/prometheus/my_key.key"
+        insecure_skip_verify: true
```

Comment:
I think it would make sense to comment this one out, and leave it as an example for the case when you cannot validate the host with a CA, WDYT?

Reply:
nice catch, thanks!